xml/install_quick.xml

```diff
@@ -101,7 +101,99 @@
        </para>
       </important>
      </listitem>
+     <listitem>
+      <para> Ports </para>
+      <para> In a &productname; cluster, internal TCP/IP ports are managed using
+       iptables controlled by Salt and so need not be manually configured.
+       However, for reference and for environments where there are existing
+       security policies, the following are the standard ports in use.</para>
+      <note>
+       <title>SSH</title>
+       <para>SSH is not strictly necessary on all nodes.</para>
+      </note>
+      <itemizedlist>
+       <listitem>
+        <para>Internal</para>
+        <itemizedlist>
+         <listitem>
+          <para>On all machines</para>
+          <itemizedlist>
+           <listitem>
+            <para>22 (ssh)</para>
+           </listitem>
+          </itemizedlist>
+         </listitem>
+         <listitem>
+          <para>On the admin node:</para>
+          <itemizedlist>
+           <listitem>
+            <para>4505-4506 (Salt)</para>
+           </listitem>
+           <listitem>
+            <para>2379 (etcd discovery)</para>
+           </listitem>
+          </itemizedlist>
+         </listitem>
+         <listitem>
+          <para>On masters and workers:</para>
```

Comment:
Having 5 different lists (for all, admin nodes, masters and workers, then masters and workers separately) makes the text incoherent. Just stick to the different 3 node types and list all the ports for the type in question, even if this means listing some ports more than once.